Form/Type/TokenType.php

```diff
@@ -0,0 +1,35 @@
+<?php
```

Comment:
Does the name of the file applies to the use? Think it should be something like TwitterType